input/docs/handbook/when-activated/index.md

```diff
@@ -44,13 +44,29 @@ In a nutshell, when object A provides a handler for an event exposed by object B
 ReactiveUI provides a variant of the Dispose pattern to help handle this concern:
 
 ```cs
-this.WhenActivated(disposables =>
+public class ActivatableControl : ReactiveUserControl<ActivatableViewModel>
 {
-    this.Bind(ViewModel, vm => vm.UserName, v => v.Username.Text)
-        .DisposeWith(disposables);
-    this.BindCommand(ViewModel, vm => vm.Login, v => v.Login)
-        .DisposeWith(disposables);
-});
+    public ActivatableControl()
+    {
+        this.InitializeComponent();
+        this.WhenActivated(disposables =>
+        {
+            // Dispose bindings to prevent dependency property memory 
```

Comment:
Might be worth putting a method like Startup() to show off it can be used for Startup logic as discussed above.